community-operators/service-binding-operator/0.0.20/service-binding-operator.v0.0.20.clusterserviceversion.yaml

```diff
@@ -18,7 +18,58 @@ spec:
       kind: ServiceBindingRequest
       name: servicebindingrequests.apps.openshift.io
       version: v1alpha1
-  description: Placeholder description
+  description: '
+  The Service Binding Operator enables users to create a managed binding between applications and operator-backed services such as databases, without having to perform manual configuration of secrets,  configmaps etc.
+  
+  ### How to use
+
+  As a cluster admin, install 2 operators into the cluster:
+   * Service Binding Operator
+   * Backing Service Operator
+
+  As a developer, 
+   * Import an application 
+   * Create Backing Service Instance
+   * Create a Service Binding request to bind the application and Backing Service e.g.:
+
+   ```
+        cat <<EOS |kubectl apply -f -
+        ---
+        apiVersion: apps.openshift.io/v1alpha1
+        kind: ServiceBindingRequest
+        metadata:
+                name: binding-request
+        spec:
+                mountPathPrefix: "/var/credentials"
+                applicationSelector:
+                        group: apps
+                        version: v1
+                        resource: deployments
+                        resourceRef: nodejs-app
+                backingServiceSelector:
+                        group: postgresql.example.dev
+                        version: v1alpha1
+                        kind: Database
+                        resourceRef: pg-instance
+                EOS
+  ```
+
+  ### Example
+
+  Check out the real-world [examples](https://github.com/redhat-developer/service-binding-operator/tree/master/examples).
```

Comment:
I'd recommend replacing line 59 with this text:

A set of examples, each of which illustrates a usage scenario for the Service Binding Operator, is being developed in parallel with the Operator. Each example includes documentation and can be run either through the OpenShift web console or command line client. The examples are available here: https://github.com/redhat-developer/service-binding-operator/blob/master/README.md#example-scenarios

Reply:
Done